### PR TITLE
feat(sprint-poker): Search GitLab issues

### LINKED ISSUE #6045: Sprint Poker: GitLab v1: Search issues
Predecessor: #6088 
Successor: https://github.com/ParabolInc/parabol/issues/6046

Like the Poker integration with Parabol tasks, the user can type into the search field. This will filter issues whose title & description don't match the search query. 

Unlike Jira or GitHub, for v1, the user cannot save search queries or add more complicated searches such as filtering by labels.

AC: 

- Create a UI that’s similar to the Parabol tasks search bar
- There’s a placeholder that says “Search issues in GitLab”
- Typing into the search field filters the issues by their title & description

### PR DESCRIPTION
**Pull request by nickoferrall**
Fix #6045 

### To test

- Integrate with GitLab. If you can't see the GitLab integration provider make sure you've added the `gitlab` feature flag. If that doesn't work, run `yarn postdeploy` to make sure the `cloudProvider` is added
- Make sure you're part of the GitLab team Parabol1
- Create a Sprint Poker meeting, go to the Scope phase, and see your GitLab issues
- Search for an issue in the search input and see that the issues filter

Note: with GitLab's search API, no results are returned if the search term doesn't match the issue title. So, if an issue is called Test and you type in "Tes", the issue may not show up.

### CODE REVIEW

**Comment by nickoferrall** [comment]
Hey @tianrunhe, just wanted to ping you here to make sure you've seen my review request 🙏

File: packages/client/clientSchema.graphql
Comment (nickoferrall) [inline packages/client/clientSchema.graphql:26]: For now, this is just a query string, but it'll be updated to an object like `GitHubSearchQuery` in a later PR when we add the project filter ids

**Review by tianrunhe — APPROVED** [review]
LGTM! Great work!

**Comment by nickoferrall** [comment]
Moving this issue to stuck until https://github.com/ParabolInc/parabol/issues/6033 makes it to Maintainer Review

File: packages/server/graphql/types/GitLabIntegration.ts
Comment (mattkrick) [inline packages/server/graphql/types/GitLabIntegration.ts:48]: -1 gitlab*

File: packages/server/graphql/types/GitLabSearchQuery.ts
Comment (mattkrick) [inline packages/server/graphql/types/GitLabSearchQuery.ts:13]: -1 i think you need to add `projectIdFilters` unless you're gonna concat those into the query string?

**Review by mattkrick — CHANGES_REQUESTED** [review]
-1 delete `sdl/resolverTypes.ts`, it's removed in master

File: packages/server/graphql/types/GitLabSearchQuery.ts
Comment (nickoferrall) [reply packages/server/graphql/types/GitLabSearchQuery.ts:13]: As we're just doing the search in this PR, I didn't include it here, but it does exist in the subsequent PR when we introduce filtering: https://github.com/ParabolInc/parabol/pull/6323

**Comment by mattkrick** [comment]
LGTM, i'll move this to stuck until the base PR gets merged

**Pull request merged**
